bootstrap/bootstrap.go

```diff
@@ -1157,11 +1171,11 @@ func (b *Bootstrap) defaultCheckoutPhase() error {
 	}
 
 	if b.Commit == "HEAD" {
-		if err := b.shell.Run("git", "checkout", "-f", "FETCH_HEAD"); err != nil {
+		if err := gitCheckout(b.shell, `-f`, `FETCH_HEAD`); err != nil {
```

Comment:
It can expose intent more to write out the long-form flags when shelling out to things; they're more revealing and more searchable in docs.

IMO short-form requires more knowledge of the reader, and are only really a when-typing optimisation - and even then, not really, because one still has to remember what they're for since they're in shell history now.